codes/options/options.py

```diff
@@ -112,7 +112,7 @@ def dict_constructor(loader, node):
             opt = yaml.load(f, Loader=Loader)
 
     opt['is_train'] = is_train
-    scale = opt['scale']
+    scale = opt.get('scale', 1)
```

Comment:
defatult to 1 or 4? Default ESRGAN is 4

Reply:
This wouldn't be used for upscaling, so a scale of anything other than 1 could be considered an error.
It's probably better to even remove the scale from the opt file and always use `1`.

Reply:
But it is true that this should probably default as 4 instead of 1, since this is also used for everything else. Perhaps before calling options.parse, force the scale to 1 in test_dvd?

Reply:
The problem is I can't not include scale in the options file or this part of the code throws a fit since it doesn't have a default. 1 makes sense in my opinion as it would be the scale for any network that doesn't actually use scale, ie this, style transfer, etc. If there's another way to do this let me know